conf/hiera.yaml

```diff
@@ -0,0 +1,11 @@
+---
+# Hiera 5 Global configuration file
+
+version: 5
+
+# defaults:
+#   data_hash: yaml_data
+# hierarchy:
+#   - name: "Global Overrides"
+#     path: "overrides.yaml"
```

Comment:
This comment might be interpreted as a description of the default for the defaults. If that's intended, then it's not correct. The default hierarchy of a Version 5 hiera.yaml is always:
```
hierarchy:
  - name: Common
    data_hash: yaml_data
```